onboarding/maintainers.md

```diff
@@ -4,6 +4,10 @@
 
 Maintainers are representatives of The Odin Project community who assist with moderating the official Discord server and the management of processes on the Github account.
 
+## Settings 
+
+- Please ensure you have 2FA authentication enabled so if a situation arises in which you need to moderate you will be able to do so. You can find the discord guide for 2FA [here](https://support.discord.com/hc/en-us/articles/219576828-Setting-up-Two-Factor-Authentication).
+
```

Comment:
Can we add that maintainers need to have 2FA setup on their Github accounts as well?